focus-android/app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -570,40 +571,19 @@ class UrlInputFragment :
         }
     }
 
-    private fun normalizeUrlAndSearchTerms(input: String): Triple<Boolean, String, String?> {
-        val isUrl = UrlUtils.isUrl(input)
-
-        val url = if (isUrl) {
-            UrlUtils.normalize(input)
-        } else {
-            SearchUtils.createSearchUrl(context, input)
-        }
-
-        val searchTerms = if (isUrl) {
-            null
-        } else {
-            input.trim { it <= ' ' }
```

Comment:
Ah nice we can skip that too because that happens via the use cases, and `SearchUrlBuilder`.